rotkehlchen/chain/ethereum/modules/yearn/utils.py

```diff
@@ -47,7 +47,7 @@ def query_yearn_vaults(db: 'DBHandler', ethereum_inquirer: 'EthereumInquirer') -
     - RemoteError
     """
     try:
-        response = requests.get(YEARN_OLD_API, timeout=DEFAULT_TIMEOUT_TUPLE)
+        response = requests.get(YEARN_OLD_API, timeout=CachedSettings().get_timeout_tuple())
```

Comment:
Out of all these I think the other thing we need is to have a default version of `requests.get()` overriden by using ourdefault timeout ... once. Not having to import this everywhere.